src/core/utils/color/ColorValue.ts

```diff
@@ -147,38 +190,120 @@ export class ColorValue {
     return scratchUInt32[0];
   }
 
-  /** Create a ColorValue from its 0xTTBBGGRR representation. */
-  public static fromTbgr(tbgr: number): ColorValue {
-    return new ColorValue(tbgr);
-  }
-
-  /** Create a ColorValue from a string representation. The following representations are supported:
+  /**
+   * Create a ColorValue from a string representation. The following representations are supported:
    * *"rgb(255,0,0)"*
    * *"rgba(255,0,0,.2)"*
    * *"rgb(100%,0%,0%)"*
    * *"hsl(120,50%,50%)"*
-   * *"#rrbbgg"*
-   * *"blanchedAlmond"* (see possible values from [[ColorByName]]). Case-insensitive.
+   * *"#rrggbb"*
    */
   public static fromString(val: string): ColorValue {
```

Comment:
Is it still needed when we have `create` that also accepts string?

Reply:
I will make private, it is used be create method.